config/vector.spec.toml

```diff
@@ -4235,6 +4235,211 @@ end
     # * type: string
     key_path = "/path/to/host_certificate.key"
 
+# Batches `log` events to LogDna's HTTP Ingestion API.
+[sinks.logdna]
+  #
+  # General
+  #
+
+  # The component type. This is a required field that tells Vector which
+  # component to use. The value _must_ be `logdna`.
+  #
+  # * required
+  # * type: string
+  # * must be: "logdna"
+  type = "logdna"
+
+  # A list of upstream source or transform IDs. See configuration for more info.
+  #
+  # * required
+  # * type: [string]
+  inputs = ["my-source-id"]
+
+  # The Ingestion API key.
+  #
+  # * required
+  # * type: string
+  api_key = "ef8d5de700e7989468166c40fc8a0ccd"
```

Comment:
:eyes:

Reply:
Its my api key but I changed it up so it should be fine :)